lib/net/Makefile

```diff
@@ -18,3 +18,6 @@ SRCS += $(LWIPSRCS) $(DRIVERSRCS)
 CFLAGS += -I $(LWIPDIR)/include
 CFLAGS += -I $(NXDK_DIR)/lib/net/nforceif/include
 CFLAGS += -I $(NXDK_DIR)/lib/net/pktdrv
+CXXFLAGS += -I $(LWIPDIR)/include
```

Comment:
Ideally we'd just have one `INCLUDES` variable that is included for both C and C++ definitions. Don't have to fix this now, but thought for the future